test/e2e/clusternetworkpolicy_test.go

```diff
@@ -0,0 +1,559 @@
+// Copyright 2020 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package e2e
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"testing"
+	"time"
+
+	log "github.com/sirupsen/logrus"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	secv1alpha1 "github.com/vmware-tanzu/antrea/pkg/apis/security/v1alpha1"
+	. "github.com/vmware-tanzu/antrea/test/e2e/utils"
+)
+
+// common for all tests.
+var (
+	allPods          []Pod
+	k8sUtils         *KubernetesUtils
+	allTestList      []*TestCase
+	pods, namespaces []string
+	podIPs           map[string]string
+	p80, p81         int
+)
+
+const (
+	// provide enough time for policies to be enforced & deleted by the CNI plugin.
+	networkPolicyDelay = 2 * time.Second
+)
+
+func failOnError(err error, t *testing.T) {
```

Comment:
We can just use `assert.NoError` for this, perhaps put `k8sUtils.Cleanup` in tests with defer to be clear.